packages/grant-explorer/src/features/projects/hooks/useApplication.ts

```diff
@@ -0,0 +1,154 @@
+import useSWR from "swr";
```

Comment:
i am assuming this is a first task and the next one we'll me moving this to datalayer

Reply:
didn't mean the hooks, but the graphql queries and co

Reply:
That's right.